VSTS/Tests/lib/Initialize-Test.ps1

```diff
@@ -3,94 +3,4 @@ param([switch]$Legacy)
 
 Write-Verbose "Importing module: TestHelpersModule"
 Import-Module $PSScriptRoot/TestHelpersModule -Verbose:$false
-Register-Mock Import-Module
```

Comment:
The 2 tasks are not using any of the mocked stuff in this file. These are mostly related to localization, which we do not do for SQ.